src/Functional/monoid.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Widmogrod\Functional;
+
+use Widmogrod\FantasyLand\Monoid;
+use Widmogrod\FantasyLand\Semigroup;
+
+const mempty = 'Widmogrod\Functional\mempty';
+
+/**
+ * mempty :: a
+ *
+ * @param Monoid $a
+ * @return Monoid
+ */
+function mempty(Monoid $a)
+{
+    return $a->getEmpty();
+}
+
+const mappend = 'Widmogrod\Functional\mappend';
+
+/**
+ * mappend :: a -> a -> a
+ *
+ * @param Semigroup $a
+ * @param Semigroup $b
+ * @return Semigroup
+ */
+function mappend(Semigroup $a, Semigroup $b)
```

Comment:
Good point.
Unfortunately in the library there are functions by names `compose` & `mcompose`.
They have different signatures.
There fore I choose tho have Haskell names.